cbqueryx/query_options.go

```diff
@@ -244,7 +244,7 @@ func (o *Options) encodeToJson() (json.RawMessage, error) {
 		m["scan_vectors"] = encodeField(o.ScanVectors)
 	}
 	if o.ScanWait > 0 {
-		m["scan_wait"] = encodeField(o.ScanWait)
+		m["scan_wait"] = encodeField(o.ScanWait.String())
```

Comment:
Rather than manually specifying .String() here, I think encodeField should have a time.Duration handler which does this?

Reply:
Done.